### ISSUE TITLE
Issue with ACWP, BCWS, BCWP, SV, CV MPP Fields methods 

### ISSUE THREAD

**Issue by amitparsha** [joniles/mpxj#340]
Hey, 

For the following MPP Fields 
ACWP: Method [getACWP()](https://www.mpxj.org/apidocs/net/sf/mpxj/Task.html#getACWP--)
BCWS: Method [getBCWS()](https://www.mpxj.org/apidocs/net/sf/mpxj/Task.html#getBCWS--)
BCWP: Method [getBCWP()](https://www.mpxj.org/apidocs/net/sf/mpxj/Task.html#getBCWP--)
SV: Method [getSV()](https://www.mpxj.org/apidocs/net/sf/mpxj/Task.html#getSV--)

These above method always return None value even there are values available in the MPP file.

CV: Method [getCV()](https://www.mpxj.org/apidocs/net/sf/mpxj/Task.html#getCV--)
CV method always return 0.0 value even there are values available in MPP file.

**Comment by joniles** [comment]
@amitparsha thanks for opening this issue. As far as I can tell, the ACWP, BCWS, BCWP, BCWP, SV and CV attributes are all calculated dynamically by MS Project rather than being stored in the MPP file. This means that I'll need to perform the same calculation in MPXJ in order to provide these values. This is certainly an enhancement I'd like to make, but it's just not something that I have prioritise currently.

I will take a look at the Project and Resource Initials attributes. I'm assuming you are referring to the `Task.getProject()` and `Task.getResourceInitials()` methods?

**Comment by amitparsha** [comment]
> @amitparsha thanks for opening this issue. As far as I can tell, the ACWP, BCWS, BCWP, BCWP, SV and CV attributes are all calculated dynamically by MS Project rather than being stored in the MPP file. This means that I'll need to perform the same calculation in MPXJ in order to provide these values. This is certainly an enhancement I'd like to make, but it's just not something that I have prioritised currently.
> 
> I will take a look at the Project and Resource Initials attributes. I'm assuming you are referring to the `Task.getProject()` and `Task.getResourceInitials()` methods?

Yes,  I'm  referring to the Task.getProject() and Task.getResourceInitials() methods.

**Comment by jayrgee** [comment]
What is the current priority on addressing the issue with ACWP, BCWS, BCWP, BCWP, SV and CV MPP fields?

**Comment by joniles** [comment]
It's something I'm looking to start work on again in the next couple of months. I've been focused on other parts of MPXJ recently. Do you have a particular subset of these attributes you need, or a deadline you are working towards?

**Comment by jayrgee** [comment]
> Do you have a particular subset of these attributes you need, or a deadline you are working towards?

I was attempting to use `getBCWP()` and `getBCWS()` methods.
